docker-entrypoint.sh

```diff
@@ -1,6 +1,37 @@
 #!/bin/bash
 set -euo pipefail
 
+# usage: file_env VAR [DEFAULT]
+#    ie: file_env 'XYZ_DB_PASSWORD' 'example'
+# (will allow for "$XYZ_DB_PASSWORD_FILE" to fill in the value of
+#  "$XYZ_DB_PASSWORD" from a file, especially for Docker's secrets feature)
+file_env() {
+	local var="$1"
+	local fileVar="${var}_FILE"
+	local def="${2:-}"
+	if [ "${!var:-}" ] && [ "${!fileVar:-}" ]; then
+		mysql_error "Both $var and $fileVar are set (but are exclusive)"
+	fi
+	local val="$def"
+	if [ "${!var:-}" ]; then
+		val="${!var}"
+	elif [ "${!fileVar:-}" ]; then
+		val="$(< "${!fileVar}")"
+	fi
+	export "$var"="$val"
+	unset "$fileVar"
+}
+
+file_env 'YOURLS_DB_HOST'
+file_env 'YOURLS_DB_USER'
+file_env 'YOURLS_DB_PASS'
+file_env 'YOURLS_DB_NAME'
+file_env 'YOURLS_DB_PREFIX'
+file_env 'YOURLS_COOKIEKEY'
```

Comment:
`YOURLS_COOKIEKEY` should be excluded.

Reply:
If it shouldn't be here, should It be removed from the readme as well? I just pulled all the env vars from this container's readme.

Reply:
Nope, it has to be documented but by default the docker script generate a random, which the best option.

Reply:
```suggestion
```